hotel/models.py

```diff
@@ -0,0 +1,207 @@
+from django.db import models
+
+from user.models import User
+#from booking.models import Booking
+
+class Category(models.Model): 
+    name = models.CharField(max_length=20)
+
+    class Meta: 
+        db_table = 'categories'
+
+class Location(models.Model): 
+    name = models.CharField(max_length=20)
+
+    class Meta: 
+        db_table = 'locations'
+
+class Hotel(models.Model): 
+    name               = models.CharField(max_length=200)
+    english_name       = models.CharField(max_length=200)
+    address            = models.CharField(max_length=200)
+    introduction       = models.CharField(max_length=200)
+    thumbnail          = models.URLField(max_length=1000)
+    category           = models.ForeignKey(Category, on_delete=models.CASCADE)
+    location           = models.ForeignKey(Location, on_delete=models.CASCADE)
+    min_people         = models.IntegerField()
+    max_people         = models.IntegerField()
+    room_count         = models.IntegerField()
+    checkin_time       = models.CharField(max_length=20)
+    checkout_time      = models.CharField(max_length=20)
+    x                  = models.DecimalField(max_digits=12, decimal_places=7)
+    y                  = models.DecimalField(max_digits=12, decimal_places=7)
```

Comment:
```suggestion
    logitude                  = models.DecimalField(max_digits=12, decimal_places=7)
    latitude                  = models.DecimalField(max_digits=12, decimal_places=7)
```
다른 개발자 혹은 사용자가 보았을 때 어떤 의미인지 알기 쉽도록 지정해주세요 :)